Tests/SwiftFusionTests/Inference/NewFactorGraphTests.swift

```diff
@@ -46,4 +46,71 @@ class NewFactorGraphTests: XCTestCase {
     // Test condition: pose 5 should be identical to pose 1 (close loop).
     XCTAssertEqual(between(x[pose1ID], x[pose5ID]).t.norm, 0.0, accuracy: 1e-2)
   }
+  
+  /// circlePose3 generates a set of poses in a circle. This function
+  /// returns those poses inside a gtsam.Values object, with sequential
+  /// keys starting from 0. An optional character may be provided, which
+  /// will be stored in the msb of each key (i.e. gtsam.Symbol).
+
+  /// We use aerospace/navlab convention, X forward, Y right, Z down
+  /// First pose will be at (R,0,0)
+  /// ^y   ^ X
+  /// |    |
+  /// z-->xZ--> Y  (z pointing towards viewer, Z pointing away from viewer)
+  /// Vehicle at p0 is looking towards y axis (X-axis points towards world y)
+  func circlePose3(numPoses: Int = 8, radius: Double = 1.0) -> Values {
```

Comment:
Could this create a `VariableAssignments` directly so that you don't have to convert it later?

Reply:
Modified to return `(hexagonId, hexagon)`, but I still feel that Frank may disagree with the current `TypedID` API as we effectively lost control of what the ID will be.